privatebin/Chart.yaml

```
@@ -5,7 +5,7 @@ description: A Helm chart for installing PrivateBin
 name: privatebin
 home: https://privatebin.info/
 icon: https://raw.githubusercontent.com/PrivateBin/assets/master/images/preview/icon.png
-version: 0.6.1
+version: 0.6.2
```

I think we should go to 0.7.0 here since we have chance at breaking configs with security context changes
```suggestion
version: 0.7.0
```